pkg/externalauthprovider/flags.go

```diff
@@ -268,7 +268,7 @@ func GetExternalAuthOptions(
 	if interactive.Enabled() && !cmd.Changed(claimMappingGroupsClaimFlag) {
 		result.claimMappingGroupsClaim, err = interactive.GetString(interactive.Input{
 			Question: "Claim mapping groups",
-			Default:  result.claimMappingGroupsClaim,
+			Default:  "groups",
```

Comment:
NP: Would it be possible to use a const here instead? 
const name like "defaultClaimMappingGroups"?